test/routes/index.spec.unit.ts

```diff
@@ -2,35 +2,35 @@ import app from "../../src/app";
 import * as request from "supertest";
 import {loadSession} from "../../src/services/redis.service";
 import {loadMockSession} from "../mock.utils";
-import Session from "../../src/session/session";
+import {COOKIE_NAME} from "../../src/properties";
 
 jest.mock("../../src/services/redis.service");
 
 const mockCacheService = (<unknown>loadSession as jest.Mock<typeof loadSession>);
 
 beforeEach(() => {
+  mockCacheService.mockRestore();
```

Comment:
mockRestore will replace the mock with it's original non-mocked implementation - https://jestjs.io/docs/en/mock-function-api#mockfnmockrestore.
maybe mockReset or mockClear might be more suitable.

Reply:
I'll check. I did experiment with a couple of things here though and this was the only thing that worked. It's the same code that we had in extensions...

Reply:
Turns out the mockRestore() call isn't required at all, so removing this line.